### PR TITLE
Client transforms for reducer data

### PR DESCRIPTION
**Pull request by sneridagh**
I have no clue where to document this feature...

- [ ] I signed and returned the [Plone Contributor Agreement](https://plone.org/foundation/contributors-agreement), and received and accepted an invitation to join a team in the Plone GitHub organization.
- [ ] I verified there aren't other open [pull requests](https://github.com/plone/volto/pulls) for the same change.
- [ ] I followed the guidelines in [Contributing to Volto](https://6.docs.plone.org/volto/contributing/index.html).
- [ ] I succesfully ran [code linting checks](https://6.docs.plone.org/volto/contributing/linting.html) on my changes locally.
- [ ] I succesfully ran [unit tests](https://6.docs.plone.org/volto/contributing/testing.html) on my changes locally.
- [ ] I succesfully ran [acceptance tests](https://6.docs.plone.org/volto/contributing/acceptance-tests.html) on my changes locally.
- [ ] If needed, I added new tests for my changes.
- [ ] If needed, I added [documentation](https://6.docs.plone.org/volto/contributing/documentation.html#narrative-documentation) for my changes, either in the Storybook or narrative documentation.
- [ ] I included a [change log entry](https://6.docs.plone.org/contributing/index.html#contributing-change-log-label) in my commits.

-----

If your pull request closes an open issue, include the exact text below, immediately followed by the issue number. When your pull request gets merged, then that issue will close automatically.

Closes #

### CODE REVIEW

**Comment by davisagli** [comment]
> I have no clue where to document this feature...

@sneridagh I guess either https://6.docs.plone.org/volto/backend/index.html or a new section in https://6.docs.plone.org/volto/blocks/index.html

File: packages/registry/news/6422.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Return empty array when `getUtilities` does not match anything @sneridagh
```
Comment (stevepiercy) [inline packages/registry/news/6422.bugfix:1]:
```suggestion
Return empty array when `getUtilities` does not match anything. @sneridagh
```

File: packages/volto/news/6422.feature [hunk 2]
```diff
@@ -0,0 +1 @@
+Introducing "client transforms for Redux reducers" @sneridagh
```
Comment (stevepiercy) [inline packages/volto/news/6422.feature:1]:
```suggestion
Introducing "client transforms for Redux reducers". @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
> > I have no clue where to document this feature...
> 
> @sneridagh I guess either https://6.docs.plone.org/volto/backend/index.html or a new section in https://6.docs.plone.org/volto/blocks/index.html

I would follow @davisagli's suggestion, as I have no clue what this thingie is!

**Comment by davisagli** [comment]
@stevepiercy It's an extension mechanism for addons to make changes to content data as it arrives from the backend.

**Comment by stevepiercy** [comment]
Gotcha. I think in [Integration with the backend](https://6.docs.plone.org/volto/backend/index.html) for sure.

However, I'm not sure how it relates to [Blocks](https://6.docs.plone.org/volto/blocks/index.html). Do we have a diagram and narrative similar to Pyramid's [Request Processing](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/router.html), where one can see where this thingie fits into the entire process? I created that diagram, and its source is available as SVG, if we want to adapt it for Plone and Volto.

**Comment by sneridagh** [comment]
@davisagli @stevepiercy I added some docs.

**Comment by sneridagh** [comment]
@stevepiercy right now it's super simple, a function in the middle of the whole pipeline. I don't know if we need further bells and whistles for now...

**Review by davisagli — APPROVED** [review]

**Pull request merged**